create-custom-map-style/README.md

```diff
@@ -0,0 +1,471 @@
+
+## How to style an existing Map from Amazon Location Service
+
+1. Maputnik is an open source visual editor for the [Mapbox Style Specification](https://www.mapbox.com/mapbox-gl-js/style-spec). At the time of writing this editor can be either used in the browser or downloaded as a binary and can run locally. We are going to use the latter option.
+
+1. Go to <https://github.com/maputnik/editor/wiki/Maputnik-CLI> and follow the instructions for installation or optionally from your terminal you can run:
+
+    ```bash
+
+    wget https://github.com/maputnik/editor/releases/download/v1.7.0/maputnik-darwin.zip
+    unzip maputnik-darwin.zip
+    chmod 755 maputnik
+    ```
+
+1. Start up Maputnik
+
+    ```bash
+
+    ./maputnik
+    ```
+
+ And open your browser to <http://localhost:8000/#12.44/40.7356/-74.0541>
+
+1. It would be best to start with a clean canvas. Click "Open" in the top navigation bar, then scroll down to the bottom of "Gallery Styles" and selected "Empty Style".
+
+    ![Open](media/MaputnikOpen.png)
+
+    *Open A Style*
+
+    ![Empty](media/MaputnikEmpty.png)
+
+    *Empty Canvas*
+
+1. Tessera is a pluggable [map tile](https://en.wikipedia.org/wiki/Tiled_web_map) server. Using the power of the [tilelive](https://github.com/mapbox/tilelive) ecosystem, it is capable of serving and rendering map tiles from many sources. To stream data from most sources you can install the tilelive providers as npm packages. For an Amazon Location Service source we use tilelive-aws.
+1. From your terminal run
+
+    ```bash
+
+    mkdir tileserver
+    cd tileserver
+    ```
+
+1. We'll need to init a new node project in the current directory
+
+    ```bash
+
+    $ npm init --yes
+
+    Wrote to ... /create-custom-map-style/tileserver/package.json:
+
+    {
+      "name": "tileserver",
+      "version": "1.0.0",
+      "description": "",
+      "main": "index.js",
+      "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+      },
+      "keywords": [],
+      "author": "",
+      "license": "ISC"
+    }
+    ```
+
+1. Next we need to add the Amazon Location Service API as a data source. However, in order to do this we need to use a local proxy that can handle SigV4 (TL;DR a way to provide your AWS credentials with http requests). For this we will use [tessera](https://github.com/mojodna/tessera) and [tilelive-aws](https://github.com/beatleboy501/tilelive-aws). From a new terminal (the `maputnik` process will be occupying the other terminal) run the following:
+
+    ```bash
+
+    nvm use 15
+    npm i tessera tilelive-aws
+    ```
+
+1. Presuming you have the standard AWS environment variables (`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`) set correctly, this should be enough to get started using `tessera`. You'll need to provide the full file path to your module, so from your terminal run:
```

Comment:
FWIW (not worth another rev), `AWS_PROFILE` is another useful environment var. When setting that, you can omit `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` **and** omit the `--profile <profile>` argument when using the CLI.